src/aiida/orm/nodes/data/code/installed.py

```diff
@@ -47,7 +47,7 @@ class Model(AbstractCode.Model):
             ...,
             title='Computer',
             description='The remote computer on which the executable resides.',
-            orm_to_model=lambda node, _: node.computer.label,
+            orm_to_model=lambda node, _: node.computer.label,  # type: ignore[attr-defined]
```

Comment:
If we know what type of label or type of `node.computer` should be, in principle can use `cast`?

`lambda node, _: cast(orm.Computer, node.computer).label`?